docs/content.zh/release-notes/flink-1.10.md

```diff
@@ -179,7 +179,7 @@ If you try to reuse your previous Flink configuration without any adjustments,
 the new memory model can result in differently computed memory parameters for
 the JVM and, thus, performance changes.
 
-Please, check [the user documentation](../deployment/memory/mem_setup.html) for more details.
+Please, check [the user documentation](https://nightlies.apache.org/flink/flink-docs-release-1.10/ops/memory/mem_setup.html) for more details.
```

Comment:
This link is specifically in the Flink 1.10 release notes. My concern is that if something gets removed or changed in a future version, the message from the release notes could be confusing because a newer page might have different content on it.